hibike/README.md

```diff
@@ -339,7 +341,35 @@ Note: These assignments are also fairly random and may not all even be
     Direction:
     BBB <-- SD
 
-7. Error Packet: Sent to indicate an error occured. 
+7. Heart Beat Request: BBB requests SD message for connectivity purposes.
```

Comment:
According to the rest of the text, both the BBB and the SD may send a heart beat request.  Please update the top line of documentation to accurately match the lines below.